src/core/Sample.cpp

```diff
@@ -170,6 +170,8 @@ void Sample::setAllPointFrames(int startFrame, int endFrame, int loopStartFrame,
 
 void Sample::playRaw(sampleFrame* dst, size_t numFrames, const PlaybackState* state, Loop loopMode) const
 {
+	if (m_buffer->size() < 1) { return; }
```

Comment:
I think this expresses the intent more clearly, but it's not that much of a big deal.
```suggestion
	if (m_buffer->size() == 0) { return; }
```